metadata-ws/src/main/resources/application.properties

```diff
@@ -50,3 +50,7 @@ swagger.apiinfo.termsOfServiceUrl=
 security.enabled=true
 security.oauth2.resource.filter-order=3
 security.oauth2.resource.jwt.key-value=-----BEGIN PUBLIC KEY-----\n@metadata-jwtKey@\n-----END PUBLIC KEY-----
+
+spring.datasource.driver-class-name=org.postgresql.Driver
```

Comment:
why do we need these values , if we already have spring.jpa.database-platform

Reply:
Liquibase documentation has this. I can verify if it works without above value.

Reply:
removed and verified